docs/index.rst

```diff
@@ -20,7 +20,8 @@ This package was originally developed at the Robert Koch Institute in the `Signa
 
    1_quickstart.rst
    2_outbreak_detection.rst
-   3_user_guide.rst
+   3_data_simulation.rst
```

Comment:
I think there could be a better name for this, as we are not simulating the data, but the "epidemic", "outbreak", ... . Maybe "Simulating epidemiological dynamics" or just "Simulation"? Is there maybe an accepted term for this in the community?